controllers/managedfusionoffering_controller.go

```diff
@@ -21,12 +21,24 @@ import (
 	"fmt"
 
 	"github.com/go-logr/logr"
+	opv1 "github.com/operator-framework/api/pkg/operators/v1"
+	opv1a1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
 	v1alpha1 "github.com/red-hat-storage/managed-fusion-agent/api/v1alpha1"
+	"github.com/red-hat-storage/managed-fusion-agent/templates"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/builder"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	controller "sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	operatorGroupName = "managed-fusion-og"
+	catalogSourceName = "managed-fusion-cat-src"
```

Comment:
```suggestion
	catalogSourceName = "managed-fusion-catsrc"
```
- just being consistent with their actual shortnames

Reply:
updated